composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "zip/magento2",
     "description": "Zip Payment Module for Magento 2",
-    "version": "1.1.11",
+    "version": "1.1.12",
```

Comment:
This should be a new major version, as it's not backwards-compatible

Reply:
:+1: Good point.

Reply:
Done